package.json

```diff
@@ -17,7 +17,7 @@
     "prepublishOnly": "yarn run build",
     "release": "4c release",
     "tdd": "jest --watch",
-    "test": "yarn lint && jest && yarn typecheck",
+    "test": "yarn lint && yarn typecheck && jest",
```

Comment:
Should run tests *after* linting

Reply:
use `4c lint` instead, it does this for you

Reply:
The lint script above does use `4c lint`, so this is still needed no?

Reply:
oh oh, woops I misread. Yeah, this makes sense